common/openapi.yml

```diff
@@ -145,6 +145,25 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/v1/datasets/{id}/export:
+    post:
+      summary: Export the dataset data into a workspace
+      tags: [ datasets ]
+      parameters:
+        - $ref: '#/components/parameters/Id'
+      operationId: exportDataset
+      requestBody:
```

Comment:
For a required argument, is there a reason to use a body instead of a path argument? It seems equivalent to me, unless the body is large and requires a multipart attachment. Using a path argument makes it a little easier to use from scripts and swagger, IMO.